docs/docs/configuration.md

```diff
@@ -39,54 +39,55 @@ Iceberg tables support table properties to configure table behavior, like the de
-| write.object-storage.partitioned-paths               | true                        | Includes the partition values in the file path                                                                                                                                                    |
-| write.data.path                                      | table location + /data      | Base location for data files                                                                                                                                                                      |
-| write.metadata.path                                  | table location + /metadata  | Base location for metadata files                                                                                                                                                                  |
-| write.delete.mode                                    | copy-on-write               | Mode used for delete commands: copy-on-write or merge-on-read (v2 only)                                                                                                                           |
-| write.delete.isolation-level                         | serializable                | Isolation level for delete commands: serializable or snapshot                                                                                                                                     |
-| write.update.mode                                    | copy-on-write               | Mode used for update commands: copy-on-write or merge-on-read (v2 only)                                                                                                                           |
-| write.update.isolation-level                         | serializable                | Isolation level for update commands: serializable or snapshot                                                                                                                                     |
-| write.merge.mode                                     | copy-on-write               | Mode used for merge commands: copy-on-write or merge-on-read (v2 only)                                                                                                                            |
-| write.merge.isolation-level                          | serializable                | Isolation level for merge commands: serializable or snapshot                                                                                                                                      |
+| Property                                             | Default                                                                                                            | Description                                                                                                                                                                                       |
```

Comment:
it would be great to not re-format the entire table when adding some new properties (this is mostly a result of IntelliJ unfortunately)